DiceKeys/Views/Screens/EditableDiceKeyView/KeyboardCommands.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  KeyboardCommands.swift
+//  DiceKeys
+//
+//  Created by Kevin Shah on 22/01/21.
+//
+
+import SwiftUI
+
+struct KeyboardCommandsModel: Identifiable {
+    var id: String
+    var key: KeyEquivalent
+}
+
+struct KeyboardCommands: Commands {
+    
+    let arrKeyboardCommandsModel: [KeyboardCommandsModel] = {
```

Comment:
This no longer needs to be a function ( { }() ) and can just a straight assignment.

Also, it could be a private let constant outside of KeyboardCommands.